src/main/sf_hud.c

```diff
@@ -177,29 +205,172 @@ void func_8008865C(void) {
 
 #pragma GLOBAL_ASM("asm/us/nonmatchings/main/sf_hud/func_800886B8.s")
 
+#ifdef IMPORT_DATA_PENDING
+void func_80088784(s32 arg0) {
+    s32 D_800D1D4C[] = {
+        0x0200B630, 0x0200A5A0, 0x02009510, 0x02008480, 0x020073F0, 0x02006360, 0x0200C6C0, 0x020052D0, 0x02004240,
+        D_20031B0,  0x02002120, 0x02001090, 0x02000000, D_2010A30,  0x020101A0, 0x0200F910, 0x0200F080, 0x0200E7F0,
+    };
+    s32 D_800D1D94[] = {
+        0x020066C0, 0x02005E30, 0x020055A0, 0x02004D10, 0x02004480, 0x02003BF0, 0x02003360,
+        0x02002AD0, 0x02002240, 0x020019B0, 0x02001120, 0x02000890, 0x02000000,
+    };
```

Comment:
```suggestion
    Gfx* D_800D1D4C[] = {
        0x0200B630, 0x0200A5A0, 0x02009510, 0x02008480, 0x020073F0, 0x02006360, 0x0200C6C0, 0x020052D0, 0x02004240,
        D_20031B0,  0x02002120, 0x02001090, 0x02000000, D_2010A30,  0x020101A0, 0x0200F910, 0x0200F080, 0x0200E7F0,
    };
    Gfx* D_800D1D94[] = {
        0x020066C0, 0x02005E30, 0x020055A0, 0x02004D10, 0x02004480, 0x02003BF0, 0x02003360,
        0x02002AD0, 0x02002240, 0x020019B0, 0x02001120, 0x02000890, 0x02000000,
    };
```
Note that these get fed into gSPDisplayList